_includes/k8s/bestpractices.md

```diff
@@ -0,0 +1,86 @@
+
+## StorageOS Pod placement
+
+StorageOS must run on all nodes that will contribute storage capacity to
+the cluster or that will host Pods which use StorageOS volumes. For production
+environments, it is recommended not to place StorageOS Pods on Master nodes.
```

Comment:
```suggestion
environments, it is not recommended to place StorageOS Pods on Master nodes.
```

recommended not sounds wrong to me.

Reply:
Hmmm the recommendation is explicitly not to do something, 'not recommended' implies that passive inaction is appropriate, which is not the case here. I'll try and think of a better way to word it.